packages/core/test/buttons/buttonTests.tsx

```diff
@@ -59,17 +59,28 @@ function buttonTestSuite(component: React.ComponentClass<any>, tagName: string)
             assert.equal(onClick.callCount, 0);
         });
 
-        it("calls onClick when enter key pressed", () => {
-            const onClick = sinon.spy();
-            button({ onClick }, true).simulate("keydown", { which: Keys.ENTER });
-            // wait for the whole lifecycle to run
-            setTimeout(() => assert.equal(onClick.callCount, 1), 0);
+        it("pressing enter triggers onKeyDown props with any modifier flags", () => {
+            checkKeyEventCallbackInvoked("onKeyDown", "keydown", Keys.ENTER);
         });
 
-        it("calls onClick when space key released", () => {
-            const onClick = sinon.spy();
-            button({ onClick }, true).simulate("keyup", { which: Keys.SPACE });
-            setTimeout(() => assert.equal(onClick.callCount, 1), 0);
+        it("pressing enter triggers onKeyUp props with any modifier flags", () => {
```

Comment:
Okay if we leave them? I guess I'm okay with deleting the tests for `onKeyUp`, but I definitely want a test in place for `onKeyDown`, since that's what broke before.